CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ All user visible changes to this project will be documented in this file. This p
 
 
 
+## master
+
+## BC Breaks
+
+- Visibility of fields `id`, `source`, `track_events_tx` and `senders` from VideoTrack and AudioTrack is changed from public to private. ([#208])
+
+### Added
+
+- `VideoTrack.id()` method for getting `VideoTrackId`. ([#208])
+- `VideoTrack.source()` method for getting `VideoSource`. ([#208])
+- `VideoTrack.add_transceiver()`, `VideoTrack.remove_transceiver()` and `VideoTrack.remove_peer()` for managing senders of `VideoTrack`. ([#208])
+- `AudioTrack.id()` method for getting `AudioTrackId`. ([#208])
+- `AudioTrack.source()` method for getting `AudioSource`. ([#208])
+- `AudioTrack.add_transceiver()`, `AudioTrack.remove_transceiver()` and `AudioTrack.remove_peer()` for managing senders of `AudioTrack`. ([#208])
```

Comment:
Those changes are not affecting public interfaces of `medea-flutter-webrtc`. Technically it's just internal refactoring of `flutter-webrtc`, because `native` crate is not published and not intended to be used as dependency in other Rust crates. 

No updates to `CHANGELOG.md` are needed for those changes.